src/Reflection/ReflectionClassConstant.php

```diff
@@ -91,6 +91,16 @@ public function getValue()
     }
 
     /**
+     * Get the value represented as a string.
+     *
+     * @return string
+     */
+    public function getValueAsString() : string
```

Comment:
@vlastavesely I see - I think @asgrim missed my comment: https://github.com/Roave/BetterReflection/pull/288#discussion_r125128644

IMO, the API should be completely removed. The consumer decides what to do with the value.